src/OrchardCore.Themes/TheBlogTheme/Views/Content-BlogPost.liquid

```diff
@@ -8,16 +8,16 @@
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                     <div class="post-heading">
-                        <h1>{{ ContentItem.Content.TitlePart.Title }}</h1>
+                        <h1>{{ Model.ContentItem.Content.TitlePart.Title }}</h1>
                         {% if fields.Subtitle.Text %}
                         <h2 class="subheading">
                             {{ fields.Subtitle.Text }}
                         </h2>
                         {% endif %}
                         <span class="meta">
                             {% assign format = "MMMM dd, yyyy" | t %}
-                            {% assign dateTime = ContentItem.CreatedUtc | date_time: format: format %}
-                            {% assign owner = "<a href=\"#\">" + ContentItem.Owner + "</a>" %}
+                            {% assign dateTime = Model.ContentItem.CreatedUtc | date_time: format: format %}
+                            {% assign owner = "<a href=\"#\">" + Model.ContentItem.Owner + "</a>" %}
```

Comment:
It's bad because the Owner value would not be encoded (security), don't use a variable here